src/egon/data/datasets/power_plants/wind_offshore.py

```diff
@@ -0,0 +1,125 @@
+
+import pandas as pd
+import geopandas as gpd
+import numpy as np
+from egon.data import db
+from shapely.geometry import Point
+
+def insert():
+    """
+    Include the off shore wind parks in egon-data
+    """
+    # Connect to the data base
+    offshore_path = 'data_bundle_egon_data/nep2035_version2021/NEP2035_V2021_scnC2035.xlsx'
```

Comment:
Please adjust the definition of the file path to remove hard-coded slashes, to achieve compatibility with windows systems. The topic was discussed in issue #398. 
You can use this as an example: https://github.com/openego/eGon-data/blob/e7e8762459870cafeae92d499ee22d94080c207c/src/egon/data/datasets/heat_supply/geothermal.py#L28-L32

Reply:
Please also add this additional source to the corresponding section in our `datasets.yml`: 
https://github.com/openego/eGon-data/blob/e7e8762459870cafeae92d499ee22d94080c207c/src/egon/data/datasets.yml#L306-L318
You can then access this entry in the datasets.yml to avoid hard coding the files' name in your code, e.g. https://github.com/openego/eGon-data/blob/212654065bb3d679a2dc1b1d8d2369ddae809674/src/egon/data/datasets/power_plants/__init__.py#L227

Reply:
Done!